web/server/www/scripts/codecheckerviewer/BugViewer.js

```diff
@@ -124,17 +124,21 @@ function (declare, domClass, dom, style, fx, Toggler, keys, on, query, Memory,
           sortMode.type = CC_OBJECTS.SortType.FILENAME;
           sortMode.ord = CC_OBJECTS.Order.ASC;
 
-          var res = CC_SERVICE.getRunResults(
-            null,
-            CC_OBJECTS.MAX_QUERY_SIZE,
-            0,
-            [sortMode],
-            reportFilter, null);
+          try {
+            var res = CC_SERVICE.getRunResults(
```

Comment:
Shouldn't we set some default values for these variables before the try blocks? What will be the value if there is an exception?
Like here and empty list for `res` or empty dictionary/map string in some other cases?